go.sum

```diff
@@ -154,6 +154,7 @@ github.com/codahale/hdrhistogram v0.9.0 h1:9GjrtRI+mLEFPtTfR/AZhcxp+Ii8NZYWq5104
 github.com/codahale/hdrhistogram v0.9.0/go.mod h1:sE/e/2PUdi/liOCUjSTXgM1o87ZssimdTWN964YiIeI=
 github.com/coreos/bbolt v1.3.2/go.mod h1:iRUV2dpdMOn7Bo10OQBFzIJO9kkE559Wcmn+qkEiiKk=
 github.com/coreos/etcd v3.3.10+incompatible/go.mod h1:uF7uidLiAD3TWHmW31ZFd/JWoc32PjwdhPthX9715RE=
+github.com/coreos/etcd v3.3.13+incompatible h1:8F3hqu9fGYLBifCmRCJsicFqDx/D68Rt3q1JMazcgBQ=
```

Comment:
Any particular reason why this file was updated ?

Reply:
This file is auto-generated by Go. I'm not sure why it added these entries here since we aren't changing dependencies that I can see, but we have a [CI rule](https://github.com/onflow/flow-go/blob/00727bd3bde6f8ca4c38363a3c1554a9757c71ab/Makefile#L135) to check that any changes are consistent with what is generated by the tooling.